src/component/Calculator.js

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import './style/Calculator.css';
+
+const Calculator = () => (
+  <div id="calculator">
+    <div className="display">
+      <span>0</span>
+    </div>
+    <div id="keypad">
+      <button type="button">AC</button>
+      <button type="button">+/-</button>
+      <button type="button">%</button>
+      <button type="button" className="orange">
+        /
+      </button>
```

Comment:
- [ ] Kindly use the correct division symbol as we have in the design: &div;. This will come in handy in later projects. If you do no have the symbol on your PC you can use this HTML entity code: `&div;`